Resources/Textures/DeltaV/Objects/Consumable/Food/hamandpineapplesalad.rsi/meta.json

```
@@ -0,0 +1,17 @@
+{
+  "version": 1,
+  "license": "CC-BY-SA-3.0",
+  "copyright": "Bowl taken from tgstation and modified by Swept at https://github.com/tgstation/tgstation/commit/40d75cc340c63582fb66ce15bf75a36115f6bdaa, Salad made by Bashie",
```

```suggestion
  "copyright": "Bowl taken from tgstation and modified by Swept at https://github.com/tgstation/tgstation/commit/40d75cc340c63582fb66ce15bf75a36115f6bdaa, Salad made by BashieAgain (Github)"
```

Or put your Discord name.